Makefile

```
@@ -30,19 +25,14 @@ GOENV=GOOS=${GOOS} GOARCH=${GOARCH} CGO_ENABLED=1 GOEXPERIMENT=boringcrypto GOFL
-
 #eg, -v
 TESTOPTS ?=
 
 DOC_BINARY := hack/documentation/document.go
 # ex -hideRules
 DOCFLAGS ?=
 
-default: all
-
-all: test build-image
+all: test osd-container-image-build
```

checkmake (0.3.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)